app/coffee/Features/Notifications/NotificationsHandler.coffee

```diff
@@ -29,14 +29,16 @@ module.exports =
 					unreadNotifications = []
 				callback(null, unreadNotifications)
 
-	createNotification: (user_id, key, templateKey, messageOpts, expiryDateTime, callback)->
+	createNotification: (user_id, key, templateKey, messageOpts, expiryDateTime, forceCreate, callback)->
 		payload = {
 				key:key
 				messageOpts:messageOpts
 				templateKey:templateKey
 		}
 		if expiryDateTime?
 			payload.expires = expiryDateTime
+		if forceCreate
+			payload.forceCreate = true
```

Comment:
not a big deal but we could just always send forceCreate = true, if nothing is there it will still be created.